### ISSUE TITLE
We cannot add code in spoilers

### ISSUE THREAD

**Issue by pierre-24** [zestedesavoir/latex-template#64]
This is not working:

```latex
\addSecret{Juste comme ça, voici la classe ingrédient, pas très bien codée :

\begin{codeBlock}{csharp}
   class Ingredient
   {
       public string Name { get; set; }
       public string Action { get; set; }
       public int Quantity { get; set; }
       public void Take()
       {
           Console.Out.WriteLine("J'ai pris " + Name);
       }

       public void UnPackage() {
           Console.Out.WriteLine(Action);
       }

       public void MesureQuantity()
       {
           Console.Out.WriteLine(Quantity);
       }

       public static Queue<Ingredient> GetRecette()
       {
           Queue<Ingredient> liste = new Queue<Ingredient>();
           liste.Enqueue(new Ingredient
           {
               Name = "farine",
               Quantity = 500,
               Action = "Rien de spécial"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "chocolat pâtissier",
               Quantity = 500,
               Action = "Faire des morceaux"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "beurre",
               Quantity = 250,
               Action = "Faire fondre"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "Cassonade",
               Quantity = 200,
               Action = "Enlever les gros morceaux"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "oeufs",
               Quantity = 2,
               Action = "Casser"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "sucre vanillé",
               Quantity = 2,
               Action = "Rien de spécial"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "sel",
               Quantity = 1,
               Action = "Prendre une pincée"

           });
           liste.Enqueue(new Ingredient
           {
               Name = "levure",
               Quantity = 1,
               Action = "Vider le sacher"

           });
           return liste;
       }
   }
\end{codeBlock}}
```

and fails with a very cryptic error:  `Paragraph ended before \FV@BeginScanning was complete \end{document}`. It is not due to the `}`, since replacing the whole code part by `x` also gives the same error.

**Comment by Karnaj** [comment]
Fixed with #65

**Comment by pierre-24** [comment]
Yeah, thanks, I forgot that I had created that issue :-°
